### PR TITLE
OctoEverywhere plugin update - Live Links Update

### PR DESCRIPTION
**Pull request by QuinnDamerell**
Updating info to include new features and Live Links!

### CODE REVIEW

File: _plugins/octoeverywhere.md [hunk 1]
```diff
@@ -79,52 +87,61 @@ compatibility:
 
 ---
 
-## 👋 Hello!
+<p class="lead">Now including OctoPrint app support, full framerate webcam streaming, live streaming, printer notifications, and much more! Your full OctoPrint portal, plugins, apps, and webcam everywhere!</b></p>
```
Comment (cp2004) [inline _plugins/octoeverywhere.md:90]:
```suggestion
<p class="lead">Now including OctoPrint app support, full framerate webcam streaming, live streaming, printer notifications, and much more! Your full OctoPrint portal, plugins, apps, and webcam everywhere!</p>
```

**Review by cp2004 — CHANGES_REQUESTED** [review]
Preview URL https://cp2004-plugins-octoprint-org-git-pr993-cp2004.vercel.app/

File: _plugins/octoeverywhere.md (hunk 1 above)
Comment (QuinnDamerell) [reply _plugins/octoeverywhere.md:90]: Good catch, thanks!

**Comment by QuinnDamerell** [comment]
Sorry if I didn't need to re-request that, I'm not sure how this works!

File: _plugins/octoeverywhere.md (hunk 1 above)
Comment (cp2004) [reply _plugins/octoeverywhere.md:90]: Marked as resolved but the changes have not been applied?

Comment (QuinnDamerell) [reply _plugins/octoeverywhere.md:90]: Oh, do I need to manually make this change? I assumed it would make it for me when I approved it?

Comment (QuinnDamerell) [reply _plugins/octoeverywhere.md]: Change made!

**Review by cp2004 — APPROVED** [review]

**Pull request merged**